ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/committer/PackageCommitter.scala

```diff
@@ -260,17 +260,13 @@ final private[kvutils] class PackageCommitter(
 
   private[this] def uploadPackages(pkgs: Map[Ref.PackageId, Ast.Package]): Either[String, Unit] =
     metrics.daml.kvutils.committer.packageUpload.preloadTimer.time { () =>
-      val errors = Graphs.topoSort(pkgs.transform((_, pkg) => pkg.directDeps)) match {
-        case Left(cycles) =>
-          List(s"cycle in package definitions: ${cycles.vertices.mkString(" -> ")}")
-        case Right(pkgIds) =>
-          pkgIds.iterator.flatMap { pkgId =>
```

Comment:
I should have filter the package ID that are not in `pkgs`.
I drop anyway all the topological sort, as it is not really usefull.